gnome-initial-setup/pages/parental-controls/gis-parental-controls-page.c

```diff
@@ -46,6 +47,24 @@ struct _GisParentalControlsPage
 
 G_DEFINE_TYPE (GisParentalControlsPage, gis_parental_controls_page, GIS_TYPE_PAGE)
 
+#define GIS_PARENTAL_CONTROLS_METRIC "5eae336c-f431-43f3-8e17-e0976b856bd8"
```

Comment:
I did have a passing thought about how we will distinguish this event from the same event submitted by something in the user session. Actually this can be done because events include the Unix UID that submitted them.

One purely bikeshed based comment based on the convention used in other projects:

```suggestion
#define GIS_PARENTAL_CONTROLS_EVENT "5eae336c-f431-43f3-8e17-e0976b856bd8"
```

Reply:
:+1: